### ISSUE TITLE
Attempted to load class "JsonMachine" from namespace "JsonMachine"

### ISSUE THREAD

**Issue by LukaJukicDev** [BlackbitDigitalCommerce/pimcore-data-director#9]
Currently, a version "<2" of the package halaxa/json-machine is used, but a few days ago a version 1.0.0 of this package was released. Here the class "JsonMachine" has been removed / replaced.

The solution would be either to use the package version 0.8 or to make the DataDirector compatible with version 1.0.0.

The following version of DataDirector was used: 2.7.9.

```
In JsonParser.php line 73:
                                                                       
  Attempted to load class "JsonMachine" from namespace "JsonMachine".  
  Did you forget a "use" statement for another namespace?       
```

**Comment by BlackbitNeueMedien** [comment]
Thanks for notifying! Just released 2.7.10 which requires helaxa/json-machine 0.8.
Will support version 1 of this library in future version (do not know yet if 2.8 or 3.0)...

**Comment by BlackbitNeueMedien** [comment]
I just implemented support for helaxa/json-machine 1.0 in data director 2.8.
